staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -166,3 +166,46 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PriorityAndFairnessConfiguration holds API Priority and Fairness parameters.
+type PriorityAndFairnessConfiguration struct {
+	metav1.TypeMeta
+
+	// WorkEstimatorConfig holds work estimator parameters.
+	WorkEstimator WorkEstimatorConfiguration
+}
+
+// WorkEstimatorConfiguration holds work estimator parameters.
+type WorkEstimatorConfiguration struct {
+	// ListWorkEstimator holds work estimator parameters related to list requests.
+	ListWorkEstimator ListWorkEstimatorConfiguration
+	// MutatingWorkEstimator holds work estimator parameters
+	// related to requests that mutate watched objects.
+	MutatingWorkEstimator MutatingWorkEstimatorConfiguration
+
+	// MaximumSeats is the maximum number of seats a request can occupy
+	MaximumSeats uint64
+}
+
+// ListWorkEstimatorConfiguration holds work estimator parameters related to list requests.
+type ListWorkEstimatorConfiguration struct {
+	// ObjectsPerSeat is a number of objects from a single list request per seat.
+	ObjectsPerSeat float64
```

Comment:
I think we should measure / estimate this, and use the measured / estimated value. I do not think it's reasonable for a cluster administrator to figure this out.